### PR TITLE
Updated voting strings for translation

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: client/src/app/site/polls/components/base-poll-detail.component.ts [hunk 1]
```diff
@@ -116,19 +116,17 @@ export abstract class BasePollDetailComponent<V extends ViewBasePoll> extends Ba
     }
 
     public async deletePoll(): Promise<void> {
-        const title = 'Delete poll';
-        const text = 'Do you really want to delete the selected poll?';
+        const title = 'Are you sure you want to delete this vote?';
```
Comment (tsiegleauq) [inline client/src/app/site/polls/components/base-poll-detail.component.ts]: Sometimes you are using translate here and sometime you are not. Stay consistent

Comment (emanuelschuetze) [reply client/src/app/site/polls/components/base-poll-detail.component.ts]: fixed.

File: client/src/app/site/motions/modules/motion-poll/motion-poll/motion-poll.component.ts [hunk 2]
```diff
@@ -123,10 +123,10 @@ export class MotionPollComponent extends BasePollComponent<ViewMotionPoll> {
     }
 
     public async deletePoll(): Promise<void> {
-        const title = 'Delete poll';
-        const text = 'Do you really want to delete the selected poll?';
+        const title = 'Are you sure you want to delete this vote?';
```
Comment (tsiegleauq) [inline client/src/app/site/motions/modules/motion-poll/motion-poll/motion-poll.component.ts]: add translate

Comment (emanuelschuetze) [reply client/src/app/site/motions/modules/motion-poll/motion-poll/motion-poll.component.ts]: Done

**Pull request merged**